.eslintignore

```diff
@@ -0,0 +1,7 @@
+/src/js/api/auth/login.test.js
```

Comment:
This seems like a bit of a hack around using `overrides` correctly, which issues did you have that could not be solved by adding override rules for Jest?

Reply:
This is because I was not able to commit because of errors like this when I was working on the code: 
![image](https://user-images.githubusercontent.com/46559129/201096198-b0c5ae38-d220-46dd-9beb-d39039cb7fb1.png)